kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -19,19 +19,13 @@ def __init__(self, pad_size, input_size, pre_pad=False):
                 size of padding to apply.
             input_size : list of 2 integers
                 size of the original signal
-            pre_pad : boolean
-                if set to true, then there is no padding, one simply adds the imaginarty part.
         """
-        self.pre_pad = pre_pad
         self.pad_size = pad_size
 
     def __call__(self, x):
-        if self.pre_pad:
-            return x
-        else:
-            paddings = [[0, 0]] * len(x.shape[:-2])
-            paddings += [[self.pad_size[0], self.pad_size[1]], [self.pad_size[2], self.pad_size[3]]]
-            return tf.pad(x, paddings, mode="REFLECT")
+        paddings = [[0, 0]] * len(x.shape[:-2])
```

Comment:
see conversation abt numpy backend padding above. Here the padding is adapted to input shape